src/styles/body-blocks/card.scss

```diff
@@ -0,0 +1,117 @@
+@import "../variables.scss";
+
+.card {
+  flex-direction: column;
+  width: 200px;
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: solid 1px #f3f3f3;
+
+  &__picture {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    margin: 0 0 4px;
+    color: $color-main-accent;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $color-secondary;
+  }
+
+  &__reviews {
+    display: flex;
+    margin-bottom: 24px;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__reviews-text {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    margin-bottom: 16px;
+    justify-content: space-between;
+  }
+
+  &__price-text {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-secondary;
+  }
+
+  &__price-amount {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $color-main-accent;
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 166px;
+    height: 40px;
+    background-color: $color-accent-blue;
+    border-radius: 5px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    font-style: normal;
+    text-transform: uppercase;
+    text-decoration: none;
+    border: 1px solid $color-accent-blue;
+    color: white;
+  }
+
+  &__button:hover {
+    background-color: #ffff;
+    color: $color-accent-blue;
+  }
+
+  @keyframes move {
+    0% {
+      size: 100%;
+    }
+    100% {
+      size: 120%;
+    }
+  }
```

Comment:
You defined `@keyframes` for `move` below. Here, it's defined incorrectly, to change element size we use `transform: scale()` 
You can read more about `size` [here](https://developer.mozilla.org/en-US/docs/Web/CSS/@page/size)